ow_materials/include/Materials.h

```diff
@@ -11,16 +11,24 @@
 namespace ow_materials
 {
 
-using MaterialID = std::uint8_t;
-
 struct Color
 {
-  double r, g, b;
+  Color(float red, float green, float blue) : r(red), g(green), b(blue) { };
```

Comment:
There's already an ignition::math::Color and Ogre::ColourValue that you could probably use here. Don't know whether it is convenient, though. I prefer ignition::math::Color just because it uses the American spelling which is consistent with the rest of our code.

Reply:
I encountered that just recently and felt tempted to switch to it. There's even some code where I have to take an ignition color and convert it to my custom color. It has an alpha channel that won't be used, but I don't that's a real problem. 

I could attempt to switch to ignition color in the next PR.